src/applications/personalization/dashboard-2/tests/unit/healthcare.unit.spec.jsx

```diff
@@ -5,18 +5,48 @@ import { expect } from 'chai';
 import { HealthCare } from '~/applications/personalization/dashboard-2/components/health-care/HealthCare';
 
 describe('HealthCare', () => {
-  describe('Manage all your prescriptions', () => {
-    const defaultProps = {
-      appointments: [],
-      authenticatedWithSSOe: true,
-      canAccessRx: false,
-      fetchConfirmedFutureAppointments: () => {},
-    };
+  const defaultProps = {
+    appointments: [],
+    authenticatedWithSSOe: true,
+    canAccessRx: false,
+    canAccessMessaging: true,
+    fetchFolder: () => {},
+    fetchConfirmedFutureAppointments: () => {},
+  };
 
-    it('when enrolled in VA health care, should not render when they have never had any prescriptions managed by the VA', () => {
-      const wrapper = mount(<HealthCare {...defaultProps} />);
-      expect(wrapper.find('Prescriptions').exists()).to.be.false;
-      wrapper.unmount();
+  describe('Prescriptions', () => {
+    describe('when enrolled in VA Health Care', () => {
+      it('should not render when they have never had any prescriptions managed by the VA', () => {
+        const wrapper = mount(<HealthCare {...defaultProps} />);
+        expect(wrapper.find('Prescriptions').exists()).to.be.false;
+        wrapper.unmount();
+      });
+    });
+  });
+
+  describe('Messaging', () => {
+    describe('when eligible for messaging', () => {
+      it('should render the unread messages count', () => {
```

Comment:
Another test you could add here is to make sure that `fetchFolder` is called.

Reply:
Done 💯